File: 백준 Study/조건문/num 2480.py
# 백준 2480번 문제 주사위 세 개 문제 
a1, a2, a3 = map(int, input().split())

if a1==a2 and a1 == a3 :
    print(10000+(a1*1000))

elif a1==a2 and a1 != a3 :
    print(1000+(a1*100))
elif a1==a3 and a1 != a2 :
    print(1000+(a1*100))
elif a2==a3 and a1 != a2 :
    print(1000+(a2*100))        

else :
    print(max(a1, a2, a3)*100)

File: 백준 Study/조건문/num2480.java
import java.util.Scanner;

public class num2480 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int input1 = sc.nextInt();
        int input2 = sc.nextInt();
        int input3 = sc.nextInt();
        sc.close();
        if(input1 == input2 && input2 == input3) {
            System.out.println(10000+(input1*1000));
        }
        else if(input1 == input2 || input2 == input3 || input1 ==input3) {
            if(input1 == input2 | input1 == input3) System.out.println(1000+(input1*100));
            else if(input2 == input3) System.out.println(1000+(input2*100));
        }
        else {
            int a = Math.max(input1, input2);
            int b = Math.max(a, input3);
            System.out.println(b*100);
        }
    }
}